core/src/main/java/discord4j/core/spec/GuildMemberEditSpec.java

```diff
@@ -0,0 +1,42 @@
+package discord4j.core.spec;
+
+import discord4j.core.object.util.Snowflake;
+import discord4j.rest.json.request.GuildMemberModifyRequest;
+
+import javax.annotation.Nullable;
+import java.util.Set;
+
+public class GuildMemberEditSpec implements Spec<GuildMemberModifyRequest> {
+
+    private final GuildMemberModifyRequest.Builder builder = GuildMemberModifyRequest.builder();
+
+    public GuildMemberEditSpec setNewVoiceChannel(@Nullable Snowflake channel) {
+        builder.channelId(channel == null ? null : channel.asLong());
+        return this;
+    }
+
+    public GuildMemberEditSpec setMute(boolean mute) {
+        builder.mute(mute);
+        return this;
+    }
+
+    public GuildMemberEditSpec setDeafen(boolean deaf) {
+        builder.deaf(deaf);
+        return this;
+    }
+
+    public GuildMemberEditSpec setNickname(@Nullable String nickname) {
+        builder.nick(nickname == null ? "" : nickname);
```

Comment:
This seems wrong. It might be that the types/annotations on the builder are wrong.

Reply:
@danthonywalker Said it was fine to convert it, difference between 3 class changes and 24 characters. I don't think it's that big of a deal. If more people can agree it should be nullable throughout I can change it.